packages/webpack-loader/src/utils.ts

```diff
@@ -1,5 +1,26 @@
 import type { Compilation as CompilationType, Compiler, sources, RuleSetRule } from 'webpack';
 
+/**
+ * Helper function to set plugin configured option on the @compiled/webpack-loader
+ *
+ * @param use
+ * @param pluginName
+ */
+const setOptionOnCompiledWebpackLoader = (use: RuleSetRule['use'], pluginName: string) => {
+  if (!use || !Array.isArray(use) || !use.length) {
```

Comment:
nit: Could this be simplified with `!use?.length || !Array.isArray(use)`?

Reply:
It could be but I think how it is now might be better for readability (i.e. checking it is an array before checking the length property)